blockchain/blockchain_test.go

```diff
@@ -848,9 +833,6 @@ func TestBlockchainInitialCandidate(t *testing.T) {
 	defer func() {
 		require.NoError(bc.Stop(context.Background()))
 	}()
-	candidate, err := sf.CandidatesByHeight(0)
```

Comment:
Please leave comment and file an issue that we will fix this test case by testing using lifeLongDelegatesProtocol to initialize the candidates

Reply:
sure